modules/apps/object/object-web/src/test/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemObjectProviderTest.java

```diff
@@ -188,19 +188,18 @@ public void testGetInfoItemProxyObjectEntryNoSuchInfoItemException()
 	public void testGetInfoItemProxyObjectEntryNullHttpServletRequest()
 		throws Exception {
 
-		ObjectEntry objectEntry = Mockito.mock(ObjectEntry.class);
-
 		String externalReferenceCode = RandomTestUtil.randomString();
+		ObjectEntry objectEntry = Mockito.mock(ObjectEntry.class);
```

Comment:
You don't need to change the order of creation of these variables (externalReferenceCode, objectEntry). The creation does not impact the assert method.

Can you keep it the way it was, please?